libs/platform/src/lib/components/form/text-area/text-area.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormModule as FdFormModule } from '@fundamental-ngx/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { TextAreaComponent } from './text-area.component';
+
+@NgModule({
+    imports: [CommonModule, FormsModule, ReactiveFormsModule, FdFormModule],
```

Comment:
We should really fix core and stop using FormModule. Its pretty expensive to import everything. Its the same as we would be importing our PLatform Module.

Reply:
Core does not have any directive for Textarea as such. Everything related to `state` and `compact` comes through the `fd-form-control` directive. We can expose the `fd-form-control` directive in a separate module, but I think it is out of the scope of this PR.

Reply:
Yes this is just a comment not something to change. The goal is to only import modules we need. FormModule imports and declares everything.